BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -775,13 +854,7 @@ extension AttributedLabel {
 
         override var accessibilityPath: UIBezierPath? {
             set { assertionFailure("cannot set accessibilityPath") }
-            get {
-                if let path = link.boundingShape.path, let container = link.container {
-                    return UIAccessibility.convertToScreenCoordinates(path, in: container)
-                }
-
-                return nil
-            }
+            get { link.boundingShape.path }
```

Comment:
I think we should cache in the local coordinate space and instead apply the `UIAccessibility.convertToScreenCoordinates` here. Similar to `UIAccessibilityEleement.AccessibilityFrameInContainerSpace`.